LINTING.md

```diff
@@ -0,0 +1,93 @@
+# Linting Standard
```

Comment:
Is this more about listing the tools, or should we be a bit more prescriptive about how they should run, like pre-commit hooks, GitHub actions, Concourse CI, etc?